nixos/modules/misc/ids.nix

```diff
@@ -300,6 +300,7 @@
       kanboard = 281;
       pykms = 282;
       kodi = 283;
+      nghttpx = 284;
```

Comment:
It does not look like this service needs to access persistent files. If so please remove static uid/gid numbers.

Reply:
Ah! Okay, I didn't realize that was one of the purposes for our static uid and gid's. I'll fixup.

Reply:
@Mic92 actually, two persistent files that it would access are the TLS certificate and key. So I think we might want to keep the static uid/gid numbers...

Reply:
For a small number of files it is better to chown at startup. Later we might use `DynamicUser` for this service.

Reply:
The ssl context is created before privileges are dropped. So uid/gid numbers are not relevant for certificates: https://github.com/nghttp2/nghttp2/blob/master/src/shrpx_worker_process.cc#L538

Reply:
Ah, I understand what you're saying now (also, TIL, thanks for pointing this out). I'll fix.